eegrasp/eegrasp.py

```diff
@@ -28,11 +30,19 @@ def __init__(self, data=None, coordinates=None, labels=None):
         Parameters
         ----------
         data: 2-d array, where the first dim are channels and the second is
-        samples.  
+        samples.
         Coordenates: ndim array with position of the electrodes.
         labels: 1-d array with the channel names.
         """
 
+        # Detect if data is a mne object
+        if isinstance(data, (BaseEpochs, BaseRaw)):
```

Comment:
I thought we were considering either `Epochs` or `Evoked`. And that each case is going to be handled differently.